### PR TITLE
Refact/keyp 312 reader api

### PR DESCRIPTION
**Pull request by jeanpierrefortune**

### CODE REVIEW

File: java/component/keyple-core/src/main/java/org/eclipse/keyple/core/seproxy/plugin/local/CardAbsentPingMonitoringJob.java [hunk 1]
```diff
@@ -93,22 +96,28 @@ public void run() {
...
             }
         };
         return job;
     }
 
+    /**
+     * (package-private)<br>
+     */
     @Override
     public void stop() {
```
Comment (pterr) [inline java/component/keyple-core/src/main/java/org/eclipse/keyple/core/seproxy/plugin/local/CardAbsentPingMonitoringJob.java:117]: Could be defined in package visibility?

File: java/component/keyple-core/src/main/java/org/eclipse/keyple/core/seproxy/plugin/local/SmartInsertionMonitoringJob.java [hunk 2]
```diff
@@ -42,32 +38,44 @@ public SmartInsertionMonitoringJob(SmartInsertionReader reader) {
...
                 }
             }
         };
     }
 
+    /**
+     * (package-private)<br>
+     */
     @Override
     public void stop() {
```
Comment (pterr) [inline java/component/keyple-core/src/main/java/org/eclipse/keyple/core/seproxy/plugin/local/SmartInsertionMonitoringJob.java:75]: package visibility

File: java/component/keyple-core/src/main/java/org/eclipse/keyple/core/seproxy/plugin/local/SmartRemovalMonitoringJob.java [hunk 3]
```diff
@@ -70,6 +71,9 @@ public void run() {
         };
     }
 
+    /**
+     * (package-private)<br>
+     */
     @Override
     public void stop() {
```
Comment (pterr) [inline java/component/keyple-core/src/main/java/org/eclipse/keyple/core/seproxy/plugin/local/SmartRemovalMonitoringJob.java:78]: package visibility

File: java/component/keyple-core/src/main/java/org/eclipse/keyple/core/seproxy/plugin/local/CardAbsentPingMonitoringJob.java (hunk 1 above)
Comment (jeanpierrefortune) [reply java/component/keyple-core/src/main/java/org/eclipse/keyple/core/seproxy/plugin/local/CardAbsentPingMonitoringJob.java]: Fixed

File: java/component/keyple-core/src/main/java/org/eclipse/keyple/core/seproxy/plugin/local/SmartInsertionMonitoringJob.java (hunk 2 above)
Comment (jeanpierrefortune) [reply java/component/keyple-core/src/main/java/org/eclipse/keyple/core/seproxy/plugin/local/SmartInsertionMonitoringJob.java]: Fixed

File: java/component/keyple-core/src/main/java/org/eclipse/keyple/core/seproxy/plugin/local/SmartRemovalMonitoringJob.java (hunk 3 above)
Comment (jeanpierrefortune) [reply java/component/keyple-core/src/main/java/org/eclipse/keyple/core/seproxy/plugin/local/SmartRemovalMonitoringJob.java]: Fixed

File: java/component/keyple-core/src/main/java/org/eclipse/keyple/core/seproxy/plugin/local/CardAbsentPingMonitoringJob.java [hunk 4]
```diff
@@ -9,12 +9,9 @@
  *
  * SPDX-License-Identifier: EPL-2.0
  ********************************************************************************/
-package org.eclipse.keyple.core.seproxy.plugin.local.monitoring;
+package org.eclipse.keyple.core.seproxy.plugin.local;
```
[the remark this answers is not in the document]
Comment (jeanpierrefortune) [reply java/component/keyple-core/src/main/java/org/eclipse/keyple/core/seproxy/plugin/local/CardAbsentPingMonitoringJob.java:12]: Done

**Pull request merged**